test/contracts/proofOfEfficiency.test.js

```diff
@@ -713,14 +713,7 @@ describe('Proof of efficiency', () => {
             sequence.timestamp,
         );
 
-        // Check the input parameters are correct
-        const circuitNextInputSC = await proofOfEfficiencyContract.getNextStarkInput(
-            newLocalExitRoot,
-            newStateRoot,
-            numBatch,
-        );
-        expect(circuitNextInputSC).to.be.equal(circuitInputSC);
-        expect(circuitNextInputSC).to.be.equal(circuitInputJS);
+        expect(circuitInpuStarktSC).to.be.equal(circuitInputStarkJS);
 
         // Check snark input
         const inputSnarkSC = await proofOfEfficiencyContract.calculateSnarkInput(
```

Comment:
Add unitary test checking values compared with JS version in https://github.com/0xPolygonHermez/zkevm-commonjs/blob/main/test/contract-utils.test.js#L16